ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -146,11 +143,12 @@ object Cli {
     opt[Path]("perf-tests-report")
       .action((inc, c) => c.copy(performanceTestsReport = Some(inc)))
       .optional()
-      .text("""The path of the the benchmark report file produced by performance tests (default: stdout).""")
+      .text(
+        "The path of the the benchmark report file produced by performance tests (default: stdout).")
 
     opt[Unit]("all-tests")
-      .action((_, c) => c.copy(allTests = true))
-      .text("""Run all default and optional tests. Respects the --exclude flag.""")
+      .text("DEPRECATED: All tests are always run by default.")
```

Comment:
I still think it's worth logging a warning if you use a deprecated option.

```suggestion
      .text("DEPRECATED: All tests are always run by default.")
      .action(_ => logger.warn("The `--all-tests` option is deprecated. All tests are always run by default."))
```

And so on for the other deprecated options.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6651/commits/69178c4e63fc29dec22b9cf744ce846a4c036a5c